packages/flutter/lib/src/animation/animations.dart

```diff
@@ -20,8 +24,13 @@ class AlwaysCompleteAnimation extends Animation<double> {
   double get value => 1.0;
 }
 
+/// An instance of an animation that is always complete.
```

Comment:
Maybe say "Using this constant involves less overhead than building an AnimationController with an initial value of 1.0. This is useful when an API expects an animation but you don't actually want to animate anything."